dali/python/nvidia/dali/external_source.py

```diff
@@ -133,6 +133,10 @@ class ExternalSource():
 
 The simplest and preferred way is to specify a `source`, which may be a callable or iterable.
 
+.. warning::
+    :meth:`nvidia.dali.ops.ExternalSource` operator doesn't support TensorFlow framework due to
+    the design limittations
```

Comment:
Maybe:
```suggestion
    :meth:`nvidia.dali.ops.ExternalSource` operator is not compatible with the TensorFlow integration. 
```
(it can also mention the reason).


Either way, it should be: `design limittations`.

Reply:
Do we want to dive that deeply and explain that ExternalSource is python based and DALI is implemented as a native C++ TF operator, and cannot execute any Python code on the demand?

Reply:
Wording suggestion accepeted.

Reply:
It's rather an implementation detail that could be too much for some people. I'd say this is fine.

I'm having second thoughts if this is clear enough that it won't work with TF.

Reply:
I would appreciate any suggestion.